tests/mixed_bag.rs

```diff
@@ -27,38 +27,40 @@ fn tests() -> Vec<Trial> {
         Trial::bench("green", |_| Err("was poisoned".into())).with_kind("kiwi"),
         Trial::bench("purple", |_| Ok(meas(100, 5))).with_ignored_flag(true),
         Trial::bench("cyan", |_| Err("not creative enough".into())).with_ignored_flag(true),
+        Trial::bench("magenta", |_| Ok(meas(5, 10))).with_ignored_flag(true).with_ignored_message(Some("flaky test!".to_string())),
```

Comment:
Can you pick another color? :D One that is at most as long as the other trial names. That way, the diffs below are way smaller.